pkg/eks/services/cloudformation.go

```diff
@@ -0,0 +1,39 @@
+package services
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+)
+
+type CloudFormationServiceInterface interface {
+	DescribeStacks(input *cloudformation.DescribeStacksInput) (*cloudformation.DescribeStacksOutput, error)
+	DeleteStack(input *cloudformation.DeleteStackInput) (*cloudformation.DeleteStackOutput, error)
+	CreateStack(input *cloudformation.CreateStackInput) (*cloudformation.CreateStackOutput, error)
+	DescribeStackEvents(input *cloudformation.DescribeStackEventsInput) (*cloudformation.DescribeStackEventsOutput, error)
+}
+
+type cloudFormationService struct {
```

Comment:
Do we want to check that `cloudFomrationService` implements the interface?

By doing something like one of these:

- `var _ CloudFormationServiceInterface = &cloudFormationService{}`
- `func NewCloudFormationService(sess *session.Session) CloudFormationServiceInterface`

Reply:
Same question applies to the other services....i won't add a comment for each.